src/utils/hooks.ts

```diff
@@ -96,7 +96,7 @@ export const useColorModeValue = (mode: SettingsContextSimpleState['mode']) => {
       border2: 'hsla(0, 0%, 0%, 0.1)',
       accent1: 'hsla(211, 100%, 60%, 1)',
       accent2: 'hsla(211, 100%, 60%, 0.08)',
-      error1: 'hsla(5, 85%, 60%, 1)',
+      error1: '#ED4747',
```

Comment:
can we stick to `hsla` format

Reply:
Though the same, I think should stick to what the designer uses in the designs. I'm also a fan of HSL, but in this case, when we have different color system on design and code, this will make things hard to maintain. The color system approach should be on designer's responsibility.